packages/common-components/src/FileInput/FileInput.tsx

```diff
@@ -143,11 +143,11 @@ const FileInput = ({
   const classes = useStyles()
   const [previews, setPreviews] = useState<any[]>([])
   const [errors, setErrors] = useState<any[]>([])
-  const [{ value }, meta, helpers] = useField<Array<FileWithPath>>(name)
+  const [{ value }, meta, helpers] = useField<Array<FileWithPath> | FileWithPath | null>(name)
```

Comment:
just for readability using `FileWithPath[]` could be cool
```suggestion
  const [{ value }, meta, helpers] = useField<FileWithPath[] | FileWithPath | null>(name)
```